src/components/Form.js

```diff
@@ -228,7 +228,8 @@ class Form extends React.Component {
                             message={this.getErrorMessage()}
                             onSubmit={this.submit}
                             onFixTheErrorsLinkPressed={() => {
-                                this.inputRefs[_.first(_.keys(this.state.errors))].focus();
+                                const focusKey = _.find(_.keys(this.inputRefs), key => _.keys(this.state.errors).includes(key));
+                                this.inputRefs[focusKey].focus();
```

Comment:
NAB 
do we need a variable here, what do you feel @ctkochan22?

Reply:
I think its fine. Arguably easier to read